tests/integration/test_remoted/test_manager_messages/test_manager_ack.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+import os
+import wazuh_testing.tools.agent_simulator as ag
+
+from time import sleep
+from wazuh_testing import remote as rd
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import FileMonitor
+
+
+# Marks
+pytestmark = pytest.mark.tier(level=1)
+
+# Variables
+current_test_path = os.path.dirname(os.path.realpath(__file__))
+test_data_path = os.path.join(current_test_path, 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_manager_ack.yaml')
+
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+# Set configuration
+parameters = [
+    {'PROTOCOL': 'tcp'},
+    {'PROTOCOL': 'udp'},
+    {'PROTOCOL': 'tcp,udp'},
+    {'PROTOCOL': 'udp,tcp'},
+]
+
+metadata = [
+    {'protocol': 'tcp'},
+    {'protocol': 'udp'},
+    {'protocol': 'tcp,udp'},
+    {'protocol': 'udp,tcp'},
+]
+
+agent_info = {
+    'manager_address': '127.0.0.1',
+    'os': 'debian7',
+    'version': '4.2.0',
+    'disable_all_modules': True
+}
+
+configuration_ids = [item['PROTOCOL'].upper() for item in parameters]
+
+# Configuration data
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+
+
+def check_manager_ack(protocol):
+    """Allow to check if the manager sends the ACK message after receiving the start-up message from agent.
```

Comment:
```suggestion
    """Checks if the manager sends the ACK message after receiving the start-up message from the agent.
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/1109/commits/09757a64f7fa8ef7cc06d07e3623064fff93f127